src/Gemfile

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 source "https://rubygems.org"
 
+gem "puma"
```

Comment:
i sorta kinda went back and forth on this. I'm half tempted to stick it in a production group or something so people can ignore it.